apps/web/src/components/settings/sections/TerminalSection.tsx

```
@@ -33,6 +49,37 @@ export function TerminalSection() {
         </p>
       </div>
 
+      {/* Editor for File Links */}
+      <div className="space-y-2">
+        <label className="text-sm font-medium text-foreground">Editor for File Links</label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)